filebeat/filebeat.full.yml

```diff
@@ -395,22 +395,206 @@ output.elasticsearch:
   # Configure curve types for ECDHE based cipher suites
   #tls.curve_types: []
 
+#------------------------------- Kafka output ---------------------------------
+#output.kafka:
+  # The list of Kafka broker addresses to connect to.
+  #hosts: ["localhost:9092"]
```

Comment:
+1.

How about "The list of Kafka broker addresses from where to fetch the cluster metadata. The cluster metadata contain the actual kafka brokers events are published to." 

Should be enough (even for docs). People knowing kafka what's meant, and people having no clue about kafka will run into issues anyway.